nomos-core/chain-defs/src/block/wire.rs

```diff
@@ -0,0 +1,102 @@
+use ark_ff::{BigInteger as _, PrimeField as _};
+use groth16::Fr;
+use num_bigint::BigUint;
+use serde::{Deserializer, Serializer};
+
+use crate::block::References;
+
+#[derive(serde::Serialize, serde::Deserialize)]
+struct WireData {
+    service_reward: Option<Vec<u8>>,
+    mempool_transactions: Vec<Vec<u8>>,
+}
+
+impl serde::Serialize for References {
```

Comment:
you could have also used the `serde_fr` module from groth16 and newtype the `Fr` in `WireData`